api/client.go

```diff
@@ -171,6 +174,7 @@ func DefaultConfig() *Config {
 	}
 
 	config.Backoff = retryablehttp.LinearJitterBackoff
+	config.CheckRetry = retryablehttp.DefaultRetryPolicy
```

Comment:
This assignment isn't necessary since you have a nil check later, and it would be good to not explicitly assign it so that anyone checking whether a func has already been set will see that it hasn't without having to see if it's been set to DefaultRetryPolicy. Just makes logic nicer in some cases.